serverscommands/instancecommands/list.go

```diff
@@ -89,58 +91,49 @@ func tableList(c *cli.Context, i interface{}) {
 		fmt.Fprintf(c.App.Writer, "Could not type assert interface\n%+v\nto []osServers.Server\n", i)
 		os.Exit(1)
 	}
-	t := tablewriter.NewWriter(c.App.Writer)
-	t.SetAlignment(tablewriter.ALIGN_LEFT)
+
 	keys := []string{"ID", "Name", "Status", "Public IPv4", "Private IPv4", "Image", "Flavor"}
-	t.SetHeader(keys)
+
+	w := new(tabwriter.Writer)
+	w.Init(c.App.Writer, 0, 8, 0, '\t', 0)
+
+	// Write the header
+	fmt.Fprintln(w, strings.Join(keys, "\t"))
 	for _, server := range servers {
```

Comment:
Alternatively, you could factor out the part that table and csv will both use into a separate function.